src/strategy/dungeons/wotlk/forgeofsouls/FosMultipliers.cpp

```diff
@@ -0,0 +1,55 @@
+#include "FosMultipliers.h"
+#include "FosActions.h"
+#include "GenericSpellActions.h"
+#include "ChooseTargetActions.h"
+#include "MovementActions.h"
+#include "FosTriggers.h"
+#include "FosActions.h"
+
+
+float BronjahmMultiplier::GetValue(Action* action) {
+    Unit* boss = nullptr;
+    boss = AI_VALUE2(Unit *, "find target", "bronjahm");
+    if (!boss)
+        return 1.0f;
+    if (botAI->IsTank(bot))
+        if (dynamic_cast<TankAssistAction*>(action))
```

Comment:
Can be combined with above line to:
`if (botAI->IsTank(bot) && dynamic_cast<TankAssistAction*>(action))`

However I don't know that you need to check IsTank(), no other character should be performing TankAssistAction, so you can probably just check that directly. Either works